### DISCUSSION TITLE
Version of Google Mobile Ads SDK

### DISCUSSION THREAD

**Discussion by Summersay415** [Shin-NiL/Godot-Android-Admob-Plugin#159] [Q&A]
Which version of Google Mobile Ads SDK used in this plugin?

**Comment by Shin-NiL** [comment]
It's using `com.google.android.gms:play-services-ads:19.3.0` right now. You can check it anytime at the file https://github.com/Shin-NiL/Godot-Android-Admob-Plugin/blob/main/config/GodotAdMob.gdap

**Reply by Momo90k** [reply]
Can i change it to the latest version in the gdap file?
